src/app/zap-templates/zcl/data-model/chip/temperature-control-cluster.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2023 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+
+  <struct name="TemperatureLevelStruct">
+    <cluster code="0x0056"/>
+    <item fieldId="0" name="Label" type="char_string" length="64" isNullable="false" writable="false"/>
+    <item fieldId="1" name="TempLevel" type="int8u" isNullable="false" writable="false"/>
+  </struct>
+
+  <cluster>
+    <name>Temperature Control</name>
+    <domain>Home &amp; Appliances</domain>
+    <description>Attributes and commands for configuring the temperature control, and reporting temperature.</description>
+    <code>0x0056</code>
+    <define>TEMPERATURE_CONTROL_CLUSTER</define>
+    <client tick="false" init="false">true</client>
+    <server tick="false" init="false">true</server>
+    <attribute side="server" code="0x0000" define="TEMP_SETPOINT" type="INT16S" min="0x8000" max="0x7fff" writable="false"  optional="true">TemperatureSetpoint</attribute>
```

Comment:
Please take out the min and max bits for all of these readonly attributes: not only do they not match the spec, we're not using the values for anything.  And having them as positive hex values for negative things is fairly confusing.

Reply:
Removed